modules/core/router/.env

```
@@ -1,4 +1,5 @@
 DOMAIN=${2:-$ROOT_DOMAIN}
```

```suggestion
DOMAIN=$ROOT_DOMAIN
```

Same for the line below. We're not reading arguments anymore inside the `.env` file.